recipes/python-3.5/meta.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: python
+  version: 3.5.1
+
+source:
+  fn: Python-3.5.1.tgz
+  url: https://www.python.org/ftp/python/3.5.1/Python-3.5.1.tgz
+  md5: be78e48cdfc1a7ad90efff146dce6cfe
+  patches:
```

Comment:
There are some interesting (read fruity) patches here. Is there any chance we can either remove them, or reference issues on the CPython bug tracker?